src/models/history.check.spec.js

```diff
@@ -0,0 +1,63 @@
+import { HistoryCheck } from "./history.check.js";
+
+describe("history.check.js", function () {
+  describe("HistoryCheck", function () {
+    describe(".manyFromSteamchartsPages adds a HistoryCheck object for each game in the gamesPagesMap.", function () {
+      describe("When the gamesPagesMap's pages properties contain one truthy and one falsy value", function () {
+        beforeEach(function () {
+          this.firstGame = { id: 1 };
+          this.secondGame = { id: 2 };
+
+          this.map = new Map();
+
+          this.map.set(this.firstGame, true);
+          this.map.set(this.secondGame, false);
+
+          this.result = HistoryCheck.manyFromSteamchartsPages(this.map);
+        });
+
+        it("the first result is an instance of HistoryCheck", function () {
```

Comment:
Add also checks for all the other properties which are set.

Imagine the HistoryCheck factory method changes how the properties source, found and checked are initialized...we would not notice it.

Same for the next method which you test.